processes/gops/process_info_linux.go

```diff
@@ -0,0 +1,20 @@
+package gops
+
+import (
+	"strings"
+
+	"github.com/shirou/gopsutil/process"
+)
+
+func pickName(p *process.Process) (string, error) {
+	cmdline, err := p.Cmdline()
+	if err != nil {
+		return "", err
+	}
+	// Assume that the process is a kernel process when it has no args
+	if strings.TrimSpace(cmdline) == "" {
```

Comment:
kernel processes/threads will be wrapped in (`[<foo>]`) square brackets - it's probably a more solid way to distinguish them.